src/Core/Utils/StdOptional.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
```

Comment:
This is not the "standard" include guard for Radium. We have to change the coding directive if we want to switch to this (simpler) guard.

Reply:
This hasn't been resolved

Reply:
@hoshiryu see: https://github.com/STORM-IRIT/Radium-Engine/pull/423/commits/6c7d2d63b6825e6a0969413ea2c85ec6c61e66f0#diff-de7544cb3809030281cea13d469fab66R9

Reply:
this must have been changed because when I open the file at the current commit, I still see `# pragma once`

Reply:
`#pragma once` offer simpler guards compatible on all plateforms.
We started to use them in Radium for some time, that's something that has already been discussed and accepted.
So I updated the documentation accordingly.

Reply:
ok, I wasn't aware of this discussion.